File: dotnet/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/MacMenu.cs
namespace AbstractFactoryDesignPattern.Example1;

public class MacMenu : Menu
{
	public override void Render()
	{
		Console.WriteLine("Rendering Mac Menu");
	}
	public override void AddItem(string item)
	{
		Console.WriteLine($"Adding '{item}' to Mac Menu");
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/src/main/java/creational/abstractfactory/example1/models/concretes/MacMenu.java
package creational.abstractfactory.example1.models.concretes;

import creational.abstractfactory.example1.models.abstracts.Menu;

public class MacMenu extends Menu {
    public void render()           { System.out.println("Rendering Mac Menu"); }
    public void addItem(String item) { System.out.println("Adding '" + item + "' to Mac Menu"); }
}
